extensions/sql-migration/config.json

```diff
@@ -1,7 +1,7 @@
 {
 	"downloadUrl": "https://github.com/Microsoft/sqltoolsservice/releases/download/{#version#}/microsoft.sqltools.migration-{#fileName#}",
 	"useDefaultLinuxRuntime": true,
-	"version": "4.10.0.9",
+	"version": "4.11.0.13",
```

Comment:
None of the changes between these two versions related to SQL NuGet. Looks safe to take this version change.